tests/platform_tests/args/advanced_reboot_args.py

```diff
@@ -84,3 +84,11 @@ def add_advanced_reboot_args(parser):
         default=False,
         help="Replace fast-reboot script on DUT",
     )
+
+    parser.addoption(
+        "--post_reboot_check_script",
+        action="store",
+        type=str,
+        default=None,
+        help="Script for checking additional states on DUT"
```

Comment:
Will you be checking in the script via a separate PR?

Reply:
Yes. The ability to designate a user specified post script is provided here.
After that, additional scripts can be provided in other PRs